lib/src/components/NxCheckbox/__tests__/NxCheckbox.test.tsx

```diff
@@ -144,4 +144,11 @@ describe('NxCheckbox', function() {
     expect(component).toHaveProp('id', 'foo');
     expect(component).toHaveProp('htmlFor', 'baz');
   });
+
+  it('pass attributes into checkbox input element', function() {
+    expect(getShallowComponent({ checkboxAttributes: { name: 'Garfield'} })
+        .find('input')).toHaveProp('name', 'Garfield');
+    expect(getShallowComponent({ checkboxAttributes: { className: 'cat', name: 'Garfield'} })
+        .find('input')).toHaveClassName('cat');
+  });
```

Comment:
I'd test what happens when something like `disabled` or the id is specified in multiple ways: in the top-level props and also in `checkboxAttributes`.

Reply:
> I'd test what happens when something like `disabled` or the id is specified in multiple ways: in the top-level props and also in `checkboxAttributes`.

Updated the tests here: 4c796beb23b05878976a76dfdcf72610774430aa